colossalai/shardformer/layer/flash_attention/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .flash_attention_opt import opt_flash_attention_forward
+
+__all__ = ['opt_flash_attention_forward']
```

Comment:
Please follow the protocol in the development notes of the `shardformer` module.

Reply:
Please read carefully https://github.com/hpcaitech/ColossalAI/tree/main/colossalai/shardformer#add-new-policy-to-shardformer

![Uploading Screenshot 2023-07-05 at 09.50.26.png…]()